R/lazyframe-utils.R

```diff
@@ -139,7 +139,7 @@ set_sink_optimizations <- function(
 #' Transforms raw percentiles into our preferred format, adding the 50th
 #' percentile.
 #' Raises an error if the percentile sequence is invalid (e.g. outside the
-#' range [0, 1]).
+#' range `[0, 1]`).
```

Comment:
Unrelated to `pl$when()`, just a quick fix because `roxygen2` 7.3.3 gave the warning:

```
✖ lazyframe-utils.R:142: @description Could not resolve link to topic "0, 1" in the dependencies or
  base packages.
ℹ If you haven't documented "0, 1" yet, or just changed its name, this is normal. Once "0, 1" is
  documented, this warning goes away.
ℹ Make sure that the name of the topic is spelled correctly.
ℹ Always list the linked package as a dependency.
ℹ Alternatively, you can fully qualify the link with a package name.
```